spark/src/main/java/org/opensearch/sql/spark/dispatcher/SparkQueryDispatcher.java

```diff
@@ -79,6 +79,9 @@ private AsyncQueryHandler getQueryHandlerForFlintExtensionQuery(
       return queryHandlerFactory.getIndexDMLHandler();
     } else if (isEligibleForStreamingQuery(indexQueryDetails)) {
       return queryHandlerFactory.getStreamingQueryHandler();
+    } else if (IndexQueryActionType.CREATE.equals(indexQueryDetails.getIndexQueryActionType())) {
+      // create should be handled by batch handler
```

Comment:
nit: This comment might be redundant. If we keep this, should we add more context ?

Reply:
will add a oneliner why this should be the case